frontend/src/v5/ui/routes/dashboard/teamspaces/projects/projectsList.component.tsx

```diff
@@ -31,40 +31,38 @@ export const ProjectsList = () => {
 
 	return (
 		<SearchContextComponent items={projects} fieldsToFilter={['name']}>
-			<Container>
```

Comment:
I think this is not used anywhere else, so you can also delete it from the `styles.ts` file